### PR TITLE
fix: update my collection relay env

### PR DESCRIPTION
**Pull request by MrSltun**
This PR resolves [] 

### Description
⚠️ THIS PR IS MERGING THE CHANGES TO 8.13.0 RELEASE CANDIDATE! ⚠️

Since the previous PR (#8777) didn't fix the issue with the insights tab not being updated, this PR updates the relay env across my collection

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- update the relay env for my collection screens - mrsltun

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Review by araujobarret — APPROVED** [review]
I guess we need to re-visit this `defaultEnvironment` again, the multiple Relay envs strikes back.

**Comment by damassi** [comment]
Can we just replace everything with `getRelayEnvironment` en-masse? Why didn't we do that to begin with? I've run into a lot of confusion here as well!

**Comment by araujobarret** [comment]
I agree with @damassi , but since we're going to cherry-pick this commit to the release branch I'd vote to do it as a follow-up PR.

**Comment by MrSltun** [comment]
@damassi @araujobarret replacing everything with `getRelayEnvironment` has already been done by Dima on #8766, I just created this PR to add a part of the change to fix the issue with the insights tab

**Comment by gkartalis** [comment]
tests should be fixed if you add [this diff](https://github.com/artsy/eigen/pull/8766/files#diff-7df6b1b964c78c45a6e6f9bb91bdcffc3b6632c957ef0294cd8cc4d52a57cd45R13-R42)

File: src/app/Scenes/Search/Search.tests.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import { act, fireEvent, screen, waitFor } from "@testing-library/react-native"
+import { Pill } from "app/Components/Pill"
```
Comment (gkartalis) [inline src/app/Scenes/Search/Search.tests.tsx:2]: 👀

Comment (MrSltun) [reply src/app/Scenes/Search/Search.tests.tsx:2]: Just repositioned, did that with Organize imports command in vscode 😅

**Review by gkartalis — APPROVED** [review]

**Comment by damassi** [comment]
oh amazing! Thanks all 👍

**Pull request merged**